scripts/xtensa-build-all.sh

```diff
@@ -233,31 +204,50 @@ do
 				XCC="none"
 				XTOBJCOPY="none"
 				XTOBJDUMP="none"
+				echo "XTENSA_TOOLS_DIR is not a directory"
 		fi
 	fi
 
 	# update ROOT directory for xt-xcc
 	if [ "$XCC" == "xt-xcc" ]
 	then
+		TOOLCHAIN=xt
 		ROOT="$XTENSA_BUILDS_DIR/$XTENSA_CORE/xtensa-elf"
 		export XTENSA_SYSTEM=$XTENSA_BUILDS_DIR/$XTENSA_CORE/config
 		PATH=$XTENSA_TOOLS_DIR/XtensaTools/bin:$OLDPATH
 	else
+		TOOLCHAIN=$HOST
 		PATH=$pwd/../$HOST/bin:$OLDPATH
 	fi
 
 	# only delete binary related to this build
 	rm -fr src/arch/xtensa/sof-$j.*
 
-	./configure --with-arch=$ARCH --with-platform=$PLATFORM \
-		--with-root-dir=$ROOT --host=$HOST --enable-debug=$BUILD_DEBUG \
-		CC=$XCC OBJCOPY=$XTOBJCOPY OBJDUMP=$XTOBJDUMP \
-		--with-dsp-core=$XTENSA_CORE $BUILD_ROM
+	rm -fr build_$j
```

Comment:
can we also include gcc or xcc in the build dir?

Reply:
I don't understand, you specify which one to use with TOOLCHAIN, it's like CROSS_COMPILE in kernel that sets tools in gcc convention (+ in our case sets some cmake specific things).

Reply:
Yes you can do that. However I'm not sure what you want to do, so maybe it would be better if you just update scripts when this buildsystem is merged.